### DISCUSSION TITLE
There is not documentation for data-normal values

### DISCUSSION THREAD

**Discussion by jhosseeth** [google/model-viewer#2772] [Q&A]
Im trying to play with the values but IDK what each field means. Its not like the camera target format $x $y $z
I need configurate the visibility for some hotspots but it takes a lot of time for me
![image](https://user-images.githubusercontent.com/73405294/132879597-6a2d0d1d-f181-4168-8108-b50cb85d8eea.png)
![image](https://user-images.githubusercontent.com/73405294/132879695-e1ff07be-0022-47d7-bc2a-d19f1e0d740e.png)

**Comment by elalish** [comment]
I think it is the same format? Here's the [documentation](https://modelviewer.dev/examples/annotations/index.html). Have you tried using our [editor's](https://modelviewer.dev/editor/) hotspot feature to add them by click? It's at the bottom of the second tab.
